Sadot/User/OrderManagement/OrderForm.cs

```diff
@@ -645,28 +651,25 @@ private void btnSwitchTable_Click(object sender, EventArgs e)
         {
             if (isExistOrder)//check if this is exist order
             {
-                if(order.TableID < 200)
```

Comment:
200 is a magic number, what is it representing?...MAX_NUM_OF_TABLES?

Reply:
Deleted - irelevant removed this logic